app/src/editor/canvas/components/ModuleHelp.jsx

```diff
@@ -1,6 +1,11 @@
+/* eslint-disable default-case */
 import React from 'react'
 import cx from 'classnames'
+import ReactMarkdown from 'react-markdown'
+import isEmpty from 'lodash/isEmpty'
+
 import * as services from '../services'
+import { createMdSnippet } from '$newdocs/components/utils'
```

Comment:
Maybe change to `import createMdSnippet from '$newdocs/utils/createMdSnippet'` for consistency?